app/javascript/react/screens/App/Overview/screens/PlanWizard/PlanWizard.js

```diff
@@ -86,11 +172,11 @@ class PlanWizard extends React.Component {
         setMigrationsFilterAction(MIGRATIONS_FILTERS.notStarted);
       }
       this.setState({
-        activeStepIndex: Math.min(activeStepIndex + 1, planWizardSteps.length - 1)
+        activeStepIndex: Math.min(activeStepIndex + 1, wizardSteps.length - 1)
       });
     } else {
       this.setState({
-        activeStepIndex: Math.min(activeStepIndex + 1, planWizardSteps.length - 1)
+        activeStepIndex: Math.min(activeStepIndex + 1, wizardSteps.length - 1)
```

Comment:
I'm sure I'm missing something, but I'm not sure I follow why we're still using the array index to track the active step if we're introducing `id`s for each step

Reply:
That's only still at the end of the logic here in the `else` case, to proceed to the next step whatever it may be, if no specific step navigation was called out. It's just to avoid "and also if you're on A go to B and if you're on B go to C" in case we hit that case.  I missed that it was in the above case though.

With this in place, I didn't have to add any extra logic here when I added the empty new step, it'll still proceed through it.